pkgs/top-level/all-packages.nix

```diff
@@ -19593,8 +19593,6 @@ in
 
   kbd = callPackage ../os-specific/linux/kbd { };
 
-  kbdKeymaps = callPackage ../os-specific/linux/kbd/keymaps.nix { };
```

Comment:
Please add an alias to aliases.nix.

Reply:
should it duplicate the information in the release notes or link to it?